cmd/gomodctl/gomodctl.go

```diff
@@ -47,6 +48,8 @@ type RootOptions struct {
 
 // Execute is exported.
 func Execute() {
+	debug.SetGCPercent(-1)
```

Comment:
Maybe a short code comment on this?

Reply:
Sure. It is from a conference I attended some time ago. Basically, if an application is a one-shot execution there is no point in making GC available and slow down the application.

https://www.dotconferences.com/2019/03/bryan-boreham-go-tune-your-memory